packages/react-examples/src/react/Layer/Layer.Hosted.Example.tsx

```diff
@@ -92,8 +96,6 @@ const styles = {
           left: '50%',
           top: '50%',
           transform: 'translate(-50%, -50%)',
-          // double quotes required to make the string show up
```

Comment:
Not sure this should be removed? Seems potentially important to help demonstrate the example's point. Also if you're removing this you should remove the entire :before styles.

Reply:
I now render the string in html (above) instead of using css  `after content`. the issue here was `I am a LayerHost with id=layerhost1"` is hard coded string and  here `layerhost1`(host id) needs to be dynamically set